Makefile

```diff
@@ -11,7 +15,11 @@ $(DEP):
 
 build: cw
 cw:
-	go build -o cw ./cmd/cw
+	go build -v -o cw -ldflags \
```

Comment:
You shouldn't need `-o cw` since we're building a package named `cw` now (IIRC).